### DISCUSSION TITLE
White label on login page

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2569] [Help with using Postal]
Hello, 

Can you tell me where to find the file to customize the login page. 

I would like something close to white label. 

Thx

**Comment by willpower232** [comment]
The view files are all HAML in the app/views/ folder.

I believe the login-specific one is https://github.com/postalserver/postal/blob/main/app/views/sessions/new.html.haml

**Reply by SaasX** [reply]
Super, thank you ;)
